domain/src/main/java/com/jakefoundation/buildabotworkshop/domain/Bot.java

```diff
@@ -0,0 +1,13 @@
+package com.jakefoundation.buildabotworkshop.domain;
+
+public interface Bot {
+    {
+        public void moveCommand(GameState);
+        public void fireCommand(GameState);
```

Comment:
The MoveCommand type and FireCommand type don't exist yet, which is why this card was closer to the bottom of the stack. To keep the project compile-ready, we need to wait to complete this card till other cards are completed.